lib/api/objectPut.js

```diff
@@ -47,6 +48,13 @@ function objectPut(authInfo, request, streamingV4Params, log, callback) {
     const requestType = 'objectPut';
     const valParams = { authInfo, bucketName, objectKey, requestType };
     const canonicalID = authInfo.getCanonicalID();
+
+    if (hasNonPrintables(objectKey)) {
+        return callback(errors.InvalidInput.customizeDescription(
+            'object keys can not contain non-printable characters',
```

Comment:
I thought you would change this one too ;)

```suggestion
            'object keys cannot contain non-printable characters',
```

Reply:
oh haha, I just did the two suggestions sorry.